npm-debug.log

```diff
@@ -0,0 +1,22 @@
+0 info it worked if it ends with ok
+1 verbose cli [ '/usr/local/Cellar/node/5.7.0/bin/node',
```

Comment:
So I tried a few different things but I'm not able to remove this file on the current PR. I guess it's because I already committed and pushed it up. I added it to the gitignore file so in the future nothing will be added from this file, but is there a way to remove what's already been pushed up? @rrgayhart